exercises/pythagorean-triplet/src/example.h

```diff
@@ -0,0 +1,18 @@
+#ifndef PYTHAGOREAN_TRIPLET
+#define PYTHAGOREAN_TRIPLET
+
+#include <stdint.h>
+#include <stdlib.h>
+
+#define LENGTH_OF_TRIPLET 3
+
+typedef struct {
+   size_t count;
+   uint16_t triplets[][LENGTH_OF_TRIPLET];
```

Comment:
Any reason you chose to use a 2D array instead of an array of structs?

Reply:
Thought array was semantically better.
In retrospect I don't think it matters that much over ease of implementation - especially for students